app/js/components/manageApps/SingleAddon.jsx

```diff
@@ -24,6 +30,21 @@ export default class SingleAddon extends React.Component{
       });
   }
 
+  handleMessageClick(event) {
+    event.preventDefault();
+    this.setState({
+      showMessageDetail: true,
+    });
+  }
+
+  hideMsgModal() {
```

Comment:
What does Msg mean?

Reply:
Have changed the naming to be more informative